README.md

```diff
@@ -35,6 +35,15 @@ Relocate-Back-End-Rails is deployed remotely via Heroku (https://relocate-back-e
 - `get '/:location/recreation/:type'`
 - `get '/:location/homeservices/:type'`
 
+- `get /favorites/:user_id`
+- `post /favorites/:user_id/:yelp_business_id (also requires "business_name" params)`
```

Comment:
What does the "also requires" part mean?

Reply:
"business name" must be sent in as a param along with the request or else it won't save